playground/backend/internal/code_processing/code_processing.go

```diff
@@ -94,18 +96,19 @@ func Process(ctx context.Context, cacheService cache.Cache, lc *fs_tool.LifeCycl
 		if err = processStep(ctxWithTimeout, pipelineId, cacheService, cancelChannel, successChannel, &compileOutput, &compileError, errorChannel, pb.Status_STATUS_COMPILE_ERROR, pb.Status_STATUS_EXECUTING); err != nil {
 			return
 		}
-	case pb.Sdk_SDK_PYTHON:
+		processSuccess(ctx, []byte(""), pipelineId, cacheService, pb.Status_STATUS_EXECUTING)
```

Comment:
Why do we need to call the `processSuccess()` method here? If I am not mistaken the `processStep()` will call it himself.

Reply:
If it is really needed please use `ctxWithTimeout` instead of `ctx` here.

Reply:
Hm, I thought somebody told me that we need to set status for Python anyway (even if don't have compile step) but if you say that processStep is enough than I remove this lines